easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -318,8 +318,29 @@ def configure_step(self, srcdir=None, builddir=None):
 
         (out, _) = run_cmd(command, log_all=True, simple=False)
 
-        return out
-
+        # sanitycheck for the configuration step
+        self.log.info("Checking python paths")
+        with open('CMakeCache.txt', 'r') as file:
+            lines = file.readlines()
+
+        # Search for Python paths in each line
+        for line in lines:
+            if line.startswith('Python3_EXECUTABLE:FILEPATH='):
+                pythonExecPath = line.split('=')[1].strip()
+            elif line.startswith('Python3_INCLUDE_DIR:FILEPATH='):
+                pythonIncludePath = line.split('=')[1].strip()
+            elif line.startswith('Python3_LIBRARY:FILEPATH='):
+                pythonLibraryPath = line.split('=')[1].strip()
+        self.log.info("Python executable path: %s", pythonExecPath)
+        self.log.info("Python include path: %s", pythonIncludePath)
+        self.log.info("Python library path: %s", pythonLibraryPath)
```

Comment:
I would only print these if they have a value

Reply:
Not only that: You could have them multiple times when you consider `Python_EXECUTABLE` and `Python3_EXECUTABLE` or the variables may be completely unset if the CMake file didn't use Python at all

Maybe create thos variables as empty lists?